packages/connect/src/device/Device.ts

```diff
@@ -337,25 +337,9 @@ export class Device extends TypedEmitter<DeviceEvents> {
             // update features
             try {
                 if (fn) {
-                    await this.initialize(
-                        !!options.useEmptyPassphrase,
-                        !!options.useCardanoDerivation,
-                    );
+                    await this.initialize(!!options.useCardanoDerivation);
                 } else {
-                    // do not initialize while firstRunPromise otherwise `features.session_id` could be affected
-                    await Promise.race([
-                        this.getFeatures(),
-                        // Edge-case: T1B1 + bootloader < 1.4.0 doesn't know the "GetFeatures" message yet and it will send no response to it
-                        // transport response is pending endlessly, calling any other message will end up with "device call in progress"
-                        // set the timeout for this call so whenever it happens "unacquired device" will be created instead
-                        // next time device should be called together with "Initialize" (calling "acquireDevice" from the UI)
-                        new Promise((_resolve, reject) =>
-                            setTimeout(
-                                () => reject(new Error('GetFeatures timeout')),
-                                GET_FEATURES_TIMEOUT,
```

Comment:
please also remove the code branches that depend on this error being thrown

Reply:
https://github.com/trezor/trezor-suite/pull/13222/commits/2094587f5dea3d1dac08610f73cb6c505a7da526